DEVELOPMENT.md

```diff
@@ -53,7 +160,7 @@ docker push gitmachete/intellij-plugin-ci
 
 ## Versioning
```

Comment:
Also, while you're at this file... pls add a section that would tell the conclusions from https://github.com/VirtusLab/git-machete-intellij-plugin/issues/460